### PR TITLE
Windows auto start

### PR DESCRIPTION
**Pull request by mattdurham**
#### PR Description 

Update changelog for RC

#### PR Checklist

- [X] CHANGELOG updated 
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,14 @@ cross-compilation issue, but will return in v0.13.0.
 
 # Main (unreleased)
 
+# 0.14.0-rc.3 (2021-04-15)
```
Comment (rfratto) [inline CHANGELOG.md:6]: Are we doing an rc 3 today?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,9 +3,14 @@ cross-compilation issue, but will return in v0.13.0.
...
+
 - [ENHANCEMENT] Add  `headers` field in `remote_write` config for Tempo. `headers`
   specifies HTTP headers to forward to the remote endpoint. (@alexbiehl)
 
+- [BUGFIX] Grafana Agent running as a Windows service should start automatically on startup
+  (@mattdurham) 
+
 # v0.13.1 (2021-04-09)
 
 - [BUGFIX] Validate that incoming scraped metrics do not have an empty label
```
Comment (rfratto) [inline CHANGELOG.md:16]: If we're cutting a release the bugfix from 0.13.1 should also be included in the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (mattdurham) [reply CHANGELOG.md:6]: Would like to so that everyone gets the auto start if they download Grafana Agent.

**Review by rfratto — APPROVED** [review]
LGTM

**Pull request merged**